.github/workflows/binaries.yml

```diff
@@ -0,0 +1,55 @@
+name: Build binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+jobs:
+  binaries:
+    name: Build lodestar binaries
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            build: |
+              npx caxa -e "dashboards/**" -e "docs/**" -D -p "yarn install --production" --input . --output "lodestar" -- "{{caxa}}/node_modules/.bin/node" "--max-old-space-size=8192" "{{caxa}}/node_modules/.bin/lodestar"
```

Comment:
We could consider adding a uncompression message, probably only worth if it takes > 3 seconds

>   -m, --uncompression-message <message>  [Advanced] A message to show when uncompressing, for example, ‘This may take a while to run the first time, please wait...’.